fluent/fluent.go

```diff
@@ -347,13 +353,26 @@ func (f *Fluent) connect() (err error) {
 }
 
 func (f *Fluent) run() {
+	drainEvents := false
+	var emitEventDrainMsg sync.Once
 	for {
+		select {
```

Comment:
I missed that `default:` is for this `select` clause...
But if so, this `select` will be processed every time in the very busy loop to drain events from `f.pending`.
Can we skip it `if drainEvents`?